pages/_app.js

```diff
@@ -1,10 +1,24 @@
+import Layout from "@/components/Layout";
 import GlobalStyle from "../styles";
+import { SWRConfig } from "swr";
+import useSWR from "swr";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
 
 export default function App({ Component, pageProps }) {
+  const { data, error, isLoading } = useSWR(
+    "https://example-apis.vercel.app/api/art",
+    fetcher
+  );
+
+  if (error) return <div>failed to load</div>;
+  if (isLoading) return <div>loading...</div>;
+
   return (
-    <>
+    <SWRConfig>
```

Comment:
you don't really need the SWRConfig here

TheSWRConfig can provide global configurations for all SWR hooks. Which can help prevent repeating yourself, if you work with several useSWR hooks.

Since you only need to fetch in your App component, there is no need for a global config.